docs/source/checks/vision/data_integrity/plot_property_label_correlation.py

```diff
@@ -15,14 +15,16 @@
 
 What is the purpose of the check?
 =================================
-The check estimates for every image property (such as brightness, contrast etc.)
-its ability to predict the label by itself.
-
-This check can help find a potential bias in the dataset, that might lead to shortcut learning - the labels being
-strongly correlated with simple image properties such as color, brightness, aspect ratio and more. This is a critical
-problem, because the model is likely to learn this property instead of the actual visual characteristics of each class,
-as it's easier to do so. In this case, the model will show high performance on images taken in similar conditions, but
-will fail in the wild, where the simple properties don't hold true.
+The check estimates for every `image property </user-guide/vision/vision_properties>`__
```

Comment:
```suggestion
The check estimates for every :doc:`image property </user-guide/vision/vision_properties>`
```

Reply:
syntactically preferable to use these direct references (easier to catch when they don't work)

Reply:
Can you please add this link also to the property_label_correlation change?